tokio-util/src/sync/cancellation_token.rs

```diff
@@ -26,7 +26,7 @@ use guard::DropGuard;
 ///
 /// ```ignore
```

Comment:
Please change this so that they are actually compiled. (There are multiple instances of this in the file.)
```suggestion
/// ```no_run
```

Reply:
Thanks for your review :)
I changed and pushed those 👍